src/openassetio-python/cmodule/src/managerApi/ManagerInterfaceBinding.cpp

```diff
@@ -119,6 +119,16 @@ struct PyManagerInterface : ManagerInterface {
                                   errorCallback);
   }
 
+  void entityTraits(const EntityReferences& entityReferences,
+                    const access::EntityTraitsAccess resolveAccess, const ContextConstPtr& context,
```

Comment:
[fix] Presumed copy/paste typo

```suggestion
                    const access::EntityTraitsAccess entityTraitsAccess, const ContextConstPtr& context,
```